docs/07-klasser.adoc

```diff
@@ -92,40 +92,41 @@ Her er Kvalitetsdimensjon og Kvalitetsdeldimensjon beskrevet samlet. dqv:Dimensi
 |Referanse|https://www.w3.org/TR/vocab-dqv/#dqv:QualityCertificate[https://www.w3.org/TR/vocab-dqv/#dqv:QualityCertificate]
 |Status|Valgfri
 |Kommentar|Klassen brukes når  man skal definere sitt eget kvalitetssertifikat.
-|Eksempel a|
-```
+|Eksempel a| [source]
+----
 :aDataset
    a dcat:Dataset ;
-   dqv:dqv:hasQualityAnnotation dqvno:isAuthoritative .
-
-der dqvno:isAuthoritative er en predefinert instans av dqv:QualityCertificate:
-
+   dqv:hasQualityAnnotation dqvno:isAuthoritative .
+----
+der `dqvno:isAuthoritative` er en predefinert instans av `dqv:QualityCertificate`:
+[souce]
+----
 dqvno:isAuthoritative
    a dqv:QualityCertificate ;
    oa:motivatedBy dqv:qualityAssessment ;
    skos:definition "kvalitetsbeskrivelse som uttrykker at noe er autoritativt"@nb , "quality description which states that something is authoritative"@en ;
    skos:prefLabel "er autoritativ"@nb , "is authoritative"@en .
-```
+----
 |===
 
 ==== Motivasjon (oa:Motivation) [[klasse-motivasjon]]
 
 [cols="30s,70"]
 |===
 |Engelsk navn|motivation
-|Beskrivelse|Brukes til å beskrive intensjonen eller motivasjonen til en Kvalitetsnote (dqv:QualityAnnotation).
+|Beskrivelse|Brukes til å beskrive intensjonen eller motivasjonen til en Kvalitetsnote (`dqv:QualityAnnotation`).
 |URI|oa:Motivation
 |Referanse|https://www.w3.org/TR/annotation-vocab/#motivation[https://www.w3.org/TR/annotation-vocab/#motivation]
 |Status|Valgfri
-|Kommentar|Den predefinerte instans https://www.w3.org/TR/vocab-dqv/#dqv:qualityAssessment[dqv:qualityAssessment] skal brukes som motivasjon for kvalitetsbeskrivelse.
+|Kommentar|Den predefinerte instans https://www.w3.org/TR/vocab-dqv/#dqv:qualityAssessment[`dqv:qualityAssessment`] skal brukes som motivasjon for kvalitetsbeskrivelse.
```

Comment:
Instansen i stedet for instans.

Reply:
Enig med deg at dobbel bestemt form er semnatisk mer korrekt å bruke her. Jeg retter opp.